api/routers.py

```
@@ -1,14 +1,30 @@
-from django.urls import path
+from django.urls import path, include
```

Ruff (0.12.2)

`django.urls.include` imported but unused

Remove unused import: `django.urls.include`

(F401)